app/controllers/reviews_controller.rb

```diff
@@ -0,0 +1,20 @@
+class ReviewsController < ApplicationController
+  def create
+    #is this the proper way to pass in params?
+
+    new_review_data = review_params
+    new_review_data[:user_id] = current_user.id
+    @vendor = Vendor.find(params[:vendor_id])
+    review = @vendor.reviews.new(new_review_data)
+    if review.save
+      flash[:notice] = ['Review added.']
+      redirect_to "/vendors/#{params[:vendor_id]}"
+    end
```

Comment:
We need an `else` statement to address what should happen if the review doesn't correctly save!